README.md

```diff
@@ -24,14 +15,118 @@ Its meetings and membership are largely focused on the direction, design, and wo
+    *  [OpenFoam](https://www.openfoam.com/)
+
+* Guidance, Navigation and Control (GNC)  
+  * Description: Packages for gnc of maritime robots.
+  * Motive: There are some control and tasks specifics to maritime robots, such as thruster control allocation, surveilance, vessels handling of tides...
+  * Repositories
+    * **[Proposal]** Thruster manager
+  * References: 
+    *  [Plankton thruster manager](https://github.com/Liquid-ai/Plankton/blob/master/uuv_control/uuv_thruster_manager/src/uuv_thrusters/thruster_manager.py)
+ 
+* Localization
+  * Description: Packages to localize maritime vehicles.
+  * Motive: Underwater vehicles have a lot of differences regarding the sensors they can use for localizing theirselves. GPS does not works underwater, as well as lidars or electromagnetic waves for long distances... But acoustic waves work pretty well, so underwater localization rely more on sensors like DVL and Sonar, that uses acoustic waves to measure distance and velocity.  
+  * Repositories:
+    * **[Proposal]**: Multilateration of acoustic beacons
+  * References: 
+    *  [Mesh Navigation](https://github.com/uos/mesh_navigation)
+    *  [Fuse Localization](https://github.com/uos/mesh_navigation)
+    *  [MagNav](https://www.gpsworld.com/us-air-force-to-explore-navigating-with-magnetism/)
+
+* Perception
+  * Description: Packages for perception algorithms for maritime robotics.
+  * Motive: Since there are different sensors such as sonars, there is a need to process this data to obtain information. Moreover, it can be created some common structures detection.
+  * Repositories:
+    * **[Proposal]**: Common Maritime Environments Detections (buoys, valves, pipes...)
+    * **[Proposal]**: Fusion of sonar and camera data
+  * References: 
+    * Suggestions...
+   
+* Robots
+  * Description: Packages with maritime robots and structures.
+  * Motive: All the software being develop shall have not only unit tests but integration tests, so robots and structures shall be developed for this purporse. These shall include vcstool for specifying packages to be used with the robot as well as ros_ign_bridge to test it on simulation.
+  * Repositories
+    * **[Proposal]** Hover type UW vehicle
+    * **[Proposal]** Torpedo shaped UW vehicle
+    * **[Proposal]** USV
+    * **[Proposal]** Intervention UW vehicle 
+  * References: 
+    *  Intervention UW sample [RexROV2](https://uuvsimulator.github.io/packages/rexrov2/intro/)
+    *  Torpedo Shaped UW vehicle [lrauv](https://github.com/osrf/lrauv/)
+   
+* Simulation
+  * Description: Documentation and index for maritime robotics Ignition plugins.
+  * Motive: There is no need to mix ros code with simulation, also there is already some maritime robotics development in Ignition, so is a good thing to contribute to ign-plugin repo instead of creating other in this organization.
+  * Repositories
+    * **[Proposal]** Maritime Robotics Ignition Plugins Index
```

Comment:
Rather than different repos we probably shouldshould maintain a table of features which maritime simulators should have and link to relevant PRs/Documentation. Something like:

| Feature | Ignition | Gazebo
| ------------- | ------------- | ------------- |
| Doppler Velocity Log |  Under Development |  [ds_sim](https://github.com/Field-Robotics-Lab/ds_sim/blob/nps_dev/gazebo_src/dsros_dvl.cc)  |
| Imaging Sonar |  Not available  |  [NPS](https://github.com/Field-Robotics-Lab/nps_uw_multibeam_sonar) |
| RADAR | Not available | Not available |
| Waves | Under Development | [Virtual RobotX](https://github.com/osrf/vrx), [ASV Wavesim](https://github.com/srmainwaring/asv_wave_sim/tree/feature/fft_waves) |  
| Acoustic Comms | Under Development | ??? |
| USBL | ??? | ??? |
| LBL | ??? | ??? |
| Ocean Currents | Upstream since fortress |  UUV, Plankton, ds_sim |
| Hydrodyamics |  | |

Theres more stuff like wind, sails, tethers that I have not included above but are important to include.

If there are ever other simulators that come along we should add them. We probably should eventually link each of these sensors to some documentation about the sensor itself.

Reply:
A bit of addition and correction to the table:
- DVL is available in all of uuv_sim, ds_sim, DAVE, and UWSim https://wiki.ros.org/uwsim
- Imaging sonar for Gazebo should be this one from uuv_sim, with additional documentation on the DAVE Wiki https://github.com/Field-Robotics-Lab/dave/wiki/image_sonar_description
  The multibeam sonar in DAVE is a very unique physics-based raytracing sonar, in contrast to the conventional sonar simulations that simulate an image. Though it also produces an image, it simulates the physics.
- USBL in Gazebo is available in DAVE https://github.com/Field-Robotics-Lab/dave/wiki/usbl_tutorial and ds_sim

Probably logical to swap the Gazebo and Ignition columns so that Gazebo, which came first, is on the left. It would resemble the Gz/Ign feature comparison page https://ignitionrobotics.org/docs/fortress/comparison

One thing I noticed is that these are just umbrella terms, and even if a simulator says it has it, it might be talking about something completely different from another simulator in terms of maturity, sophistication, and functionality. That kind of information would best be captured outside the table, if that's something this document wants to capture.

Alternatively, this table could be captured in a different document, since this is the charter document and requires a lot of approvals to revise in the future.

Reply:
Hi :) I registered for the presentation about the Imaging sonar. As @mabelzhang describes, the [multibeam sonar plugin at Dave](https://github.com/Field-Robotics-Lab/nps_uw_multibeam_sonar) output sonar image but it's quite a different one.